### PR TITLE
Fix some code

### PR DESCRIPTION
**Pull request by erdenezul**
@batamar

### CODE REVIEW

File: src/data/resolvers/queries/insights.ts [hunk 1]
```diff
@@ -503,13 +494,14 @@ const insightQueries = {
       // team members gather
       const fixedChartData = await fixChartData(userData.chartDatas, 'date', 'count');
       const user = await Users.findOne({ _id: userData._id });
-      userData.chartDatas.map(row => {
+      userData.chartDatas.forech(row => {
```
Comment (batamar) [inline src/data/resolvers/queries/insights.ts]: forEach

File: src/data/resolvers/queries/aggregationUtils.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Converts dateField into string using timeFormat
+ * @param fieldName
+ * @param timeFormat
+ */
+export const getDateFieldAsStr = async ({
```
Comment (batamar) [inline src/data/resolvers/queries/aggregationUtils.ts]: Remove async

File: src/data/resolvers/queries/aggregationUtils.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  fieldName = '$createdAt',
+  timeFormat = '%Y-%m-%d',
+}): Promise<{
```
Comment (batamar) [inline src/data/resolvers/queries/aggregationUtils.ts]: Remove Promise, Just {$dateToString: {
    format: string;
    date: string;
  }

Comment (erdenezul) [reply src/data/resolvers/queries/aggregationUtils.ts]: done

**Pull request merged**